public/static/docs/glossary.js

```diff
@@ -8,13 +8,12 @@ export default {
       name: 'Workspace',
       match: ['workspace'],
       desc: `
-Directory containing all your project files. For example raw datasets, source
-code, ML models, etc. A workspace becomes a **DVC project** when
-[\`dvc init\`](/doc/command-reference/init) is run, and
-[DVC-files](/doc/user-guide/dvc-file-format) or stage files are created in it.
-
-Note that [external outputs](/doc/user-guide/managing-external-data) also
-form part of your expanded workspace, technically.
+Collection of all your project files e.g. raw datasets, sourc code, ML models,
```

Comment:
source

Reply:
I like the previous way better. My take on this - we complicate this for no good reason.

Reply:
OK 😞 reverted.

But I think the part abut "A workspace becomes a DVC project..." is unnecessary.

Reply:
yeah, it's not perfect for sure, along with DVC Project, but let's save for regular updates/next iteration